vault.yaml

```diff
@@ -52,13 +53,37 @@ pipeline:
   - runs: |
       install -m644 -D "./vault.confd" "${{targets.destdir}}/etc/conf.d/vault"
 
-      install -m755 -D ./vault/bin/vault "${{targets.destdir}}/usr/sbin/vault"
-      install -m644 -D "./vault.hcl" "${{targets.destdir}}/etc/vault/vault.hcl"
+      install -m755 -D ./vault/bin/vault "${{targets.destdir}}/usr/bin/vault"
+      setcap cap_ipc_lock=+ep  "${{targets.destdir}}/usr/bin/vault"
```

Comment:
Do we need to setcap after strip?

Reply:
Yes, testing that change at the moment

Reply:
Haven't actually got this working locally - not sure if it's because melange or apko are out of date?

Reply:
I've pushed the update to check if it works here...